saskatoon/member/forms.py

```diff
@@ -209,3 +212,21 @@ def save(self):
         instance.save()
 
         return instance
+
+
+class PasswordChangeForm(auth_forms.PasswordChangeForm):
+
+    def __init__(self, user, *args, **kwargs):
+        super().__init__(user, *args, **kwargs)
+
+        # replace widgets with placeholder values to fit Notika theme
+        self.fields['old_password'].widget = PasswordInput(attrs={'class': 'form-control', 'autocomplete': 'current-password', 'placeholder': 'Old password', 'autofocus': True })
+        self.fields['new_password1'].widget = PasswordInput(attrs={'class': 'form-control', 'autocomplete': 'new-password', 'placeholder': 'New password'})
+        self.fields['new_password2'].widget = PasswordInput(attrs={'class': 'form-control', 'autocomplete': 'new-password', 'placeholder': 'Confirm password'})
```

Comment:
can you try and set the `widgets` field instead? see [OrganizationForm](https://github.com/LesFruitsDefendus/saskatoon-ng/blob/9eee8ffba13f0978916fab800ec3970be38a8737/saskatoon/member/forms.py#L138) for example

Reply:
`widgets` field is inherited from `ModelForm`. this class extends `django.contrib.auth.PasswordForm` which very lightly extends `BaseForm`.

If for legibility, I could try to simplify with a `map`?

Reply:
ok all good